trio/_tools/gen_exports.py

```diff
@@ -105,42 +102,78 @@ def create_passthrough_args(funcdef: ast.FunctionDef | ast.AsyncFunctionDef) ->
     return "({})".format(", ".join(call_args))
 
 
-def run_linters(file: File, source: str) -> str:
-    """Run isort and black on the specified file, returning the new source.
+def run_black(file: File, source: str) -> tuple[bool, str]:
+    """Run black on the specified file, returning tuple of success and result string.
 
     :raises ImportError: If black is not installed
-    :raises SystemExit: If either failed.
-    """
-    # imported to check that `subprocess` calls to black will succeed
+    :raises SystemExit: If black failed."""
```

Comment:
nit: I'm not sure docstring styling but maybe want to put the `"""` end on a newline.

Reply:
Also this no longer raises.

Reply:
Updated docstring in 18e76680

Reply:
Put docstring end on newline in 60fbe57a